.github/workflows/installation.yml

```diff
@@ -0,0 +1,110 @@
+name: Installation
+on: [push]
+
+jobs:
+  linux-vm:
+    name: ${{ matrix.python-version }} - ${{ matrix.method }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version:
+          - '2.7'
+          - '3.6'
+          - '3.7'
+        method:
+          - 'python ./install.py'
+          - 'pip install --target /opt/rez .'
+        include:
+          - method: 'python ./install.py'
+            exports: 'PATH=${PATH}:/opt/rez/bin/rez'
+          - method: 'pip install --target /opt/rez .'
+            exports: 'PATH=${PATH}:/opt/rez/bin PYTHONPATH=${PYTHONPATH}:/opt/rez'
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@master
+
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Install
+      run: |
+        ${{ matrix.method }}
+        find /opt/rez/ -maxdepth 2
+
+    - name: rez status
+      run: |
+        export ${{ matrix.exports }}
+        rez status
+
+    - name: rez-pip --install .
+      run: |
+        export ${{ matrix.exports }}
+        rez-pip --install .
+        rez view rez
+
+    - name: Import rez package in Python
+      run: |
+        export ${{ matrix.exports }}
+
+        # Still needed as there's no fallback to use system's python
+        rez bind python
+
+        echo "Checking rez as python package: ==========================================="
+        rez env rez -- python -c 'import rez;print(rez.__file__)'
+
+  linux-containers:
+    name: python:${{ matrix.python-version }} - ${{ matrix.method }}
+    runs-on: ubuntu-latest
+    container: python:${{ matrix.python-version }}
```

Comment:
That runs the test in a docker container right? What's the reason for doing this? I can't think of a case where install rez would fail in a container but not on a bare metal/vm machine.

Reply:
I was just double checking that if the system already has Python it wasn't doing anything different from using [actions/setup-python](https://github.com/actions/setup-python)

It didn't, [see here](https://github.com/j0yu/bleeding-rez/runs/621905284?check_suite_focus=true), so I'll remove it